global.json

```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
       "version": "8.0.101",
-      "rollForward": "latestFeature"
+      "rollForward": "latestMajor"

```

Comment:
This allows main to be able to use net9.0 sdk if this is the only sdk installed. And this PR fixes the warnings discovered by this new SDK (syntax and vulnerabilities).